common/app/views/support/Commercial.scala

```diff
@@ -23,7 +24,7 @@ object Commercial {
   def shouldShowAds(page: Page): Boolean = page match {
     case c: model.ContentPage if c.item.content.shouldHideAdverts => false
     case p: model.Page if p.metadata.sectionId == "identity" => false
-    case s: model.SimplePage if s.metadata.contentType == "Signup" => false
+    case s: model.SimplePage if s.metadata.contentType.contains(Signup) => false
```

Comment:
Noticed this was comparing unrelated types!

Reply:
Can [Scalactic](http://www.scalactic.org/) help avoid such issues? According to the docs:
> A compiler error for an equality comparison that would always yield false looks like:
> ```scala
> import TypeCheckedTripleEquals._
> Some("hi") === "hi"
> error: types Some[String] and String do not adhere to the type constraint
> selected for the === and !== operators; the missing implicit parameter is
> of type org.scalactic.Constraint[Some[String],String]
>           Some("hi") === "hi"